labman/gui/test/test_plate.py

```diff
@@ -6,6 +6,7 @@
 # The full license is in the file LICENSE, distributed with this software.
 # ----------------------------------------------------------------------------
 
+# from json import dumps
```

Comment:
Can you uncomment these tests? They will get executed in travis, so we can see there if they pass or not - and if not, we should be able to see the error.